package.json

```diff
@@ -11,6 +11,7 @@
     "type": "git",
     "url": "https://github.com/jonschlinkert/window-size.git"
   },
+	"bin": "cli.js",
```

Comment:
would you mind using 2 spaces to be consistent with the rest of the code? other than that, looks great! thanks!